crates/models/gptneox/src/lib.rs

```diff
@@ -43,6 +43,34 @@ pub struct GptNeoX {
     _mmap: Option<Mmap>,
 }
 
+// feed-forward network
+fn _gpt_neox_ff(layer: &Layer, context: &ggml::Context, input: &Tensor) -> Tensor {
```

Comment:
Remove underscore, move to the end of the file (the public-facing stuff should come first)